neo4j/src/test/java/org/hibernate/ogm/datastore/neo4j/test/mapping/UnidirectionalManyToOneTest.java

```diff
@@ -50,35 +53,28 @@ public void prepareDb() throws Exception {
 
 	@Test
 	public void testMapping() throws Exception {
-		assertNumberOfNodes( 3 );
-		assertRelationships( 2 );
-
-		String jugNode = "(jug:JUG:ENTITY {jug_id: {jug}.jug_id, name: {jug}.name })";
-		String emmanuelNode = "(e:Member:ENTITY {member_id: {e}.member_id, name: {e}.name})";
-		String jeromeNode = "(j:Member:ENTITY {member_id: {j}.member_id, name: {j}.name})";
-
-		Map<String, Object> jugProperties = new HashMap<String, Object>();
-		jugProperties.put( "jug_id", jug.getId() );
-		jugProperties.put( "name", jug.getName() );
-
-		Map<String, Object> emmanuelProperties = new HashMap<String, Object>();
-		emmanuelProperties.put( "member_id", emmanuel.getId() );
-		emmanuelProperties.put( "name", emmanuel.getName() );
-
-		Map<String, Object> jeromeProperties = new HashMap<String, Object>();
-		jeromeProperties.put( "member_id", jerome.getId() );
-		jeromeProperties.put( "name", jerome.getName() );
-
-		Map<String, Object> params = new HashMap<String, Object>();
-		params.put( "jug", jugProperties );
-		params.put( "e", emmanuelProperties );
-		params.put( "j", jeromeProperties );
-
-		assertExpectedMapping( "jug", jugNode, params );
-		assertExpectedMapping( "e", emmanuelNode, params );
-		assertExpectedMapping( "j", jeromeNode, params );
-		assertExpectedMapping( "r", jugNode + " - [r:memberOf] - " + emmanuelNode, params );
-		assertExpectedMapping( "r", jugNode + " - [r:memberOf] - " + jeromeNode, params );
+		NodeForGraphAssertions jugNode = node( "jug", JUG.class.getSimpleName(), ENTITY.name() )
```

Comment:
FWIW, I'd prefer a style more closely to Cypher, without any parameters etc.:

    assertThatNodeExists("(jug:JUG:ENTITY {jug_id: summer_camp, name: JUG Summer Camp })");
    assertThatRelationshipExists("(jug:JUG:ENTITY {jug_id: summer_camp} - [r:memberOf] - > ...)");

Much more readable IMO. Also more reliable as you actually write/read the expected persistent form.

But no need to change from my side, just to share my view :)